lib/pry-byebug/cli.rb

```diff
@@ -1 +0,0 @@
-require 'pry-byebug'
```

Comment:
Yes, I can confirm that this branch re-introduces that bug. I'll restore `cli.rb` but have it not load `pry`.